src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -2065,13 +2067,19 @@ def cli(self,interface="",output=None):
                 continue
 
             # Helper address is not set
-            p5 = re.compile(r'^Helper +address +is +(?P<address>[\w\.\:\s]+)$')
+            p5 = re.compile(r'^Helper +address +is +not +set$')
             m = p5.match(line)
             if m:
-                if 'not set' not in m.groupdict()['address']:
-                    interface_dict[interface]['helper_address'] = \
-                        m.groupdict()['address']
                 continue
+
+            # Helper address is 10.1.1.1
+            p5_0 = re.compile(r'^Helper +address +is +(?P<address>[\d\.]+)$')
+            m = p5_0.match(line)
+            if m:
+                interface_dict[interface]['helper_address'] = \
+                    m.groupdict()['address']
```

Comment:
`helper_address` must be a list according to the schema. This case should be a list with a single value:
```suggestion
                    [m.groupdict()['address']]
```